app/routes.py

```diff
@@ -1,2 +1,232 @@
-from flask import Blueprint
+from app import db
+from app.models.task import Task
+from flask import request, Blueprint, make_response
+from flask import jsonify
+from sqlalchemy import asc, desc
+import time
+import datetime
+import requests
+from flask import current_app as app
+import os 
+from app.models.goal import Goal
+
+path = "https://slack.com/api/chat.postMessage"
+
+tasks_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+
+@tasks_bp.route("", methods=["POST", "GET"], strict_slashes=False)
+def tasks():
```

Comment:
Small readability suggestion, it could be good to break up these large functions into smaller functions.  So one function to handle the GET request and the other handles the POST request.